pom.xml

```diff
@@ -519,6 +519,27 @@
         <version>${plugin.deploy.version}</version>
       </plugin>
 
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
```

Comment:
I think this is a build-time only plugin so we don't need to report the license for this?